receivers/slack/slack.go

```diff
@@ -517,16 +525,15 @@ func initialCommentForImage(alert *types.Alert) string {
 	sb.WriteString(", ")
 
 	sb.WriteString("*Labels*: ")
-
-	var n int
-	for k, v := range alert.Labels {
-		sb.WriteString(string(k))
-		sb.WriteString(" = ")
-		sb.WriteString(string(v))
-		if n < len(alert.Labels)-1 {
-			sb.WriteString(", ")
-			n++
+	if len(alert.Labels) == 0 {
+		sb.WriteString("None")
+	} else {
+		lstrs := make([]string, 0, len(alert.Labels))
```

Comment:
NOTE: The previous logic here is incorrect IMO as it's using all labels including private ones. I will fix this in a followup to use `templates.ExtendedAlert` which has sanitized labels.

Ex:

`
Firing: always firing prom (copy), Labels: __grafana_autogenerated__ = true, __grafana_receiver__ = Both Slack Types, __grafana_route_settings_hash__ = 554a8065f06504ff, alertname = always firing prom (copy), __alert_rule_uid__ = cedgbhztxup6oe, prom = true, grafana_folder = Grafana Common
`